common/build/eslint-config-fluid/package.json

```
@@ -31,6 +31,7 @@
     "@rushstack/eslint-plugin-security": "~0.2.5",
     "@typescript-eslint/eslint-plugin": "~5.9.0",
     "@typescript-eslint/parser": "~5.9.0",
+    "eslint-config-prettier": "^8.5.0",
```

```suggestion
    "eslint-config-prettier": "~8.5.0",
```

tilde ranges are generally safer for lint-related deps.